libqtile/widget/volume.py

```diff
@@ -221,7 +224,9 @@ def get_volume(self):
             check_mute = subprocess.getoutput(self.check_mute_command)
 
         if self.check_mute_string in check_mute:
-            return -1
+            self.mute = 1
+        else:
+            self.mute = 0
```

Comment:
Can you use python's True + False for these literals?

Reply:
Better yet, can you make `get_volume()` return a tuple that's the volume and whether or not the widget is muted? You can override it in your test with some monkey patching.